packages/gatsby-recipes/src/cli.js

```diff
@@ -538,31 +517,76 @@ module.exports = ({ recipe, graphqlPort, projectRoot }) => {
         return <Error width="100%" state={state} />
       }
 
+      const staticMessages = {}
+      for (let step = 0; step < state.context.currentStep; step++) {
+        staticMessages[step] = [
+          {
+            type: `mdx`,
+            key: `mdx-${step}`,
+            value: state.context.stepsAsMdx[step],
+          },
+        ]
+      }
+      lodash.flattenDeep(state.context.stepResources).forEach((res, i) => {
+        staticMessages[res._currentStep].push({
+          type: `resource`,
+          key: `finished-stuff-${i}`,
+          value: res._message,
+        })
+      })
+
+      log(`staticMessages`, staticMessages)
+
+      if (isDone) {
+        process.nextTick(() => {
+          subscriptionClient.close()
+          exit()
+          process.stdout.write(`\n\n---\n\n\nThe recipe finished successfully!\n\n`)
```

Comment:
Looks like the robots want it formatted this way?

```suggestion
          process.stdout.write(
            `\n\n---\n\n\nThe recipe finished successfully!\n\n`
          )
```